.github/workflows/code-checker.yaml

```diff
@@ -111,6 +111,39 @@ jobs:
           fi
           brew install boost libomp ninja tbb ccache
           make check-proto-product && make verifiers
+  centos:
+    name: Code Checker CentOS 7
+    # Run in centos docker 
+    runs-on: ubuntu-18.04
```

Comment:
why still ubuntu-18.04

Reply:
because action runner only have ubuntu .. and our centos 7 build run inside the centos 7 docker. It does not matter